From 648ca7103439e44e7f83035fa701994a408f8d01 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Wed, 13 May 2009 16:46:58 +0000
Subject: Continue to poll when power supply device is marked unknown. Might fix rh#495493

---
diff --git a/src/dkp-supply.c b/src/dkp-supply.c
index 7a5e8e5..cb6db00 100644
--- a/src/dkp-supply.c
+++ b/src/dkp-supply.c
@@ -548,7 +548,8 @@ dkp_supply_refresh (DkpDevice *device)
 		 * if we are charging or discharging */
 		g_object_get (device, "state", &state, NULL);
 		if (state == DKP_DEVICE_STATE_CHARGING ||
-		    state == DKP_DEVICE_STATE_DISCHARGING)
+		    state == DKP_DEVICE_STATE_DISCHARGING ||
+		    state == DKP_DEVICE_STATE_UNKNOWN)
 			supply->priv->poll_timer_id =
 				g_timeout_add_seconds (DKP_SUPPLY_REFRESH_TIMEOUT,
 						       (GSourceFunc) dkp_supply_poll_battery, supply);
--
cgit v0.8.2
